fixtures/showtec/explorer-250-wash-pro.json

```diff
@@ -0,0 +1,684 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Explorer 250 Wash",
+  "categories": ["Color Changer", "Moving Head"],
+  "meta": {
+    "authors": ["Luke Nelson"],
+    "createDate": "2023-04-29",
+    "lastModifyDate": "2023-04-29",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2023-03-25",
+      "comment": "created by Q Light Controller Plus (version 4.12.6)"
+    }
+  },
+  "links": {
+    "manual": [
+      "https://www.manualslib.com/manual/837961/Showtec-Explorer-250-Wash.html",
+      "https://www.highlite.com/media/attachments/MANUAL/40178_MANUAL_D_V1.pdf"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=_E7AQH2ugoM"
+    ]
+  },
+  "physical": {
+    "dimensions": [379, 476, 428],
+    "weight": 33.5,
+    "power": 350,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "MSD 250",
+      "lumens": 6700
+    },
+    "lens": {
+      "name": "Fresnel",
+      "degreesMinMax": [8, 22]
+    }
+  },
+  "wheels": {
+    "Color Wheel": {
+      "slots": [
+        {
+          "type": "Open"
+        },
+        {
+          "type": "Color",
+          "name": "Red",
+          "colors": ["#ff0000"]
+        },
+        {
+          "type": "Color",
+          "name": "Blue",
+          "colors": ["#0000ff"]
+        },
+        {
+          "type": "Color",
+          "name": "Green",
+          "colors": ["#00ff00"]
+        },
+        {
+          "type": "Color",
+          "name": "CTF 3200K",
+          "colorTemperature": "3200K"
+        },
+        {
+          "type": "Color",
+          "name": "CTF 5300K",
+          "colorTemperature": "5300K"
+        },
+        {
+          "type": "Color",
+          "name": "UV",
+          "colors": ["#8000ff"]
+        },
+        {
+          "type": "Gobo",
+          "name": "Rainbow Effect"
+        }
```

Comment:
I think this should not be a separate gobo, but instead a Gobo wheel rotation capability.